parsec/core/logged_core.py

```diff
@@ -116,6 +116,10 @@ async def get_user_info(self, user_id: UserID) -> UserInfo:
             created_on=user_certif.timestamp,
         )
 
+    async def get_user_info_from_device_id(self, device_id: DeviceID) -> UserInfo:
```

Comment:
The DeviceID class has a user_id property. You don't need the `get_user_info_from_device_id` method and the `split()`, you can just call `get_user_info(creator.user_id)`

Reply:
thanks for the tip